src/d3d9/d3d9_device.cpp

```diff
@@ -4060,7 +4070,7 @@ namespace dxvk {
     desc.IsAttachmentOnly   = TRUE;
     desc.IsLockable         = Lockable;
 
-    if (FAILED(D3D9CommonTexture::NormalizeTextureProperties(this, D3DRTYPE_TEXTURE, &desc)))
+    if (FAILED(D3D9CommonTexture::NormalizeTextureProperties(this, D3DRTYPE_SURFACE, &desc)))
```

Comment:
Nice find actually. We don't have a lot of resource type based checks in NormalizeTextureProperties, so I don't worry too much that this will cause regressions.

Reply:
It didn't affect anything really, since there were no D3DRTYPE_SURFACE type validations in NormalizeTextureProperties before, but there is one now for plain surfaces, so that's how I spotted this bit.